### ISSUE TITLE
Linking a case as a contact of another case

### ISSUE THREAD

**Issue by Iheanacho2027** [SORMAS-Foundation/SORMAS-Project#1858]
### Situation Description
There are times where already created suspect cases are later confirmed to be contacts of already confirmed cases. At the moment, what is done on SORMAS is that users have to delete the already created suspect case, create a contact and convert to a case, which is a duplication of effort and unnecessary use of time
### Feature Description
We should have a feature whereby a case can be linked as a contact of another case after case creation, the user will only be required to enter the name of the case its linked to, the type of contact, date of last contact and relationship to case @bernardsilenou @Chinedar @foyiri @hzi-old @MartinWahnschaffeSymeda

**Comment by foyiri** [comment]
Yes we had this discussion on how we can transfer contact from one case
to another case while it remains in the other case where it transfer from

**Comment by Iheanacho2027** [comment]
@foyiri no, i mean making an already created case to be a contact of another case, not moving a contact from one case to another

**Comment by bernardsilenou** [comment]
when you latter realized that a  case already created was a contact to another case,  I suggest we do not delete that case already. We should create a new case and when creating the contacts of the new case, we should choose the person of the former case as the contact person of the new case.
@Iheanacho2027 @foyiri do you agree with this ?

**Comment by Iheanacho2027** [comment]
Sorry i am not clear on this @bernardsilenou, pls can you rephrase

**Comment by MateStrysewskeSym** [comment]
@bernardsilenou I think the issue with that approach is that the newly created contact will not have the case set as the resulting case. The solution for this problem is probably simply to allow users to pick already existing cases as the resulting case for contacts if they want to.

**Comment by Iheanacho2027** [comment]
We can have a link named "make case contact of an existing confirmed case", and the user will be presented with the default contact creation form linked to that confirmed case after clicking the link

**Comment by bernardsilenou** [comment]
@MateStrysewskeSym  I now understand you. The resulting case ID will be missing in the contact table. Can we not automatically fill this previous case id as the resulting case id on the contact table?
Ii is good that users should be able to pick a resulting case for a contact. In this situation, we will need to consider the time duration  in incubation period, disease, etc for validation
Maybe we discuss this in next sprint. My general opinion is to find a solution that we can automatically link them based on certain rules.

**Comment by Iheanacho2027** [comment]
Pls can we prioritize this @MateStrysewskeSym

**Comment by Iheanacho2027** [comment]
@bernardsilenou something similar was done for event as shown in the image below. There should be a button with caption "Link as contact of confirmed case"

![image](https://user-images.githubusercontent.com/50739609/89125933-bc474300-d4d9-11ea-8fee-e2ac4becd156.png)
